packages/accounts/src/msca/multi-owner-account.ts

```diff
@@ -26,11 +28,14 @@ import { TokenReceiverPlugin } from "./plugins/token-receiver/plugin.js";
 export const createMultiOwnerMSCASchema = <
   TTransport extends SupportedTransports = Transport
 >() =>
-  createBaseSmartAccountParamsSchema<TTransport>().extend({
-    owner: SignerSchema,
-    index: z.bigint().optional().default(0n),
-    excludeDefaultTokenReceiverPlugin: z.boolean().optional().default(false),
-  });
+  createBaseSmartAccountParamsSchema<TTransport>()
+    .omit({ owner: true })
+    .extend({
+      signer: SignerSchema,
+      owners: z.array(zAddress).default([]),
```

Comment:
would be good to add validation check if the address is sorted.

Reply:
yea but then it's no different than a developer calling the contract directly and getting an error. We can handle this on our end and hide that implementation detail from devs